kolibri/plugins/learn/assets/src/views/SearchFiltersPanel/SelectGroup.vue

```diff
@@ -216,6 +216,17 @@
 
 <style lang="scss" scoped>
 
+  // Do not let text oveflow in select menu
+  /deep/ .ui-select-content {
+    width: 100%;
+
+    // remove position absolute to prevent text from getting under button
```

Comment:
Thanks again for the helpful explanations, this will surely be valuable in the future